pkg/reconciler/subscription/subscription.go

```diff
@@ -58,8 +58,8 @@ const (
 )
 
 var (
-	v1beta1ChannelGVK = v1beta1.SchemeGroupVersion.WithKind("Channel")
-	v1ChannelGVK      = v1.SchemeGroupVersion.WithKind("Channel")
+	//v1ChannelGVK = v1.SchemeGroupVersion.WithKind("Channel")
```

Comment:
oops :)

Reply:
thanks for catching, I had it staged but forgot to push that change cleaning up